src/resources/2010/CrownOfThrones.ts

```diff
@@ -329,6 +329,13 @@ export const ridingFamiliars: readonly FamiliarRider[] = [
   },
 ] as const;
 
+const FULL_RIDING_LIST = [
+  ...ridingFamiliars,
+  ...Familiar.all()
+    .filter((f) => ridingFamiliars.some(({ familiar }) => familiar === f))
```

Comment:
Is this meant to be an inverted condition?

Reply:
yes